Examples/RKSearchExample/RKSearchExample.xcodeproj/project.pbxproj

```diff
@@ -338,7 +338,7 @@
 				GCC_WARN_ABOUT_RETURN_TYPE = YES;
 				GCC_WARN_UNINITIALIZED_AUTOS = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 6.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 8.1;
```

Comment:
We use RestKit on iOS8.0 and above.  Is there anyway we can make the deployment target 8.0 instead of 8.1?